code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -1183,7 +1186,7 @@
 	category = list(
 		RND_CATEGORY_MECHFAB_CYBORG_MODULES + RND_SUBCATEGORY_MECHFAB_CYBORG_MODULES_SERVICE
 	)
-	
+
```

Comment:
put whitespace back

Reply:
I'm not really sure what this is, nothing else has more spaces here and i think VS did that change itself 
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/22140677/267cf7c1-6835-4995-81b3-e3872191a1ba)

Reply:
oh it's upstream, never mind